readme.md

```diff
@@ -73,6 +83,12 @@ Values: `start` `middle` `end`
 
 Position to truncate the string.
 
+##### space
+
+Type: `boolean`
+
+Add space to ellipsis. Depends on the `position` option.
```

Comment:
This description could be improved. You're not adding space **to** the ellipsis, you're adding space **between** the text and the ellipsis.

Reply:
I updated the description.